pkg/install/v1alpha1/cstor_volume.go

```diff
@@ -574,7 +574,7 @@ spec:
     Calculate the replica count
     Add as many poolUid to resources as there is replica count
     */}}
-    {{- $poolUids := keys .ListItems.cvolPoolList.pools }}
+    {{- $poolUids := keys .ListItems.cvolPoolList.pools | randomize }}
```

Comment:
Can we make sure that we take only `Healthy` replicas ? We might not want to take those pools which are not `Healthy`?

Reply:
no.. doesn't need to be healthy.. and, there can be many other conditions on deciding the pool, which are not in the scope of this PR

Reply:
Ok